specification/caas-published-cohort-definitions.yaml

```diff
@@ -27,6 +27,188 @@ info:
     To be completed.
     ## Onboarding
     To be completed.
+
```

Comment:
We need to add some negative tests as well

Reply:
<img width="901" alt="image" src="https://github.com/NHSDigital/caas-published-cohort-definitions/assets/120651954/0737ea98-8cf6-439d-a50f-eb62a100ea20">

negative tests added